waspc/waspc.cabal

```diff
@@ -6,7 +6,7 @@ cabal-version: 2.4
 --    Consider using hpack, or maybe even hpack-dhall.
 
 name:           waspc
-version:        0.10.4
+version:        0.10.6
```

Comment:
What happened to 0.10.5?

Reply:
Let's go with `0.10.5` since that's the next release's version. We tend to group features under a version until we release them.

Latest released version is `0.10.4`, so let's do this under `0.10.5`

Reply:
🤷‍♂️ when I merged it, it said 0.10.5 so I bumped it thinking it auto deploys main 😅, will change to 0.10.5